### ISSUE TITLE
Download Button

### ISSUE THREAD

**Issue by Acovendesigns** [underscoretype/fontsampler-js#15]
Would be really nice to have an optional free download button built into the grid.

**Comment by kontur** [comment]
Thanks for the feature request.

I am actually refactoring the module a bit to allow better variable font instantiation, and as part of that I am thinking of adding a feature that would allow adding any referenced DOM node as part of the UI. Something like:

```
var custom = document.getElementById("mycustomelement")
options = {
  order: ["fontsize", "lineheight", custom, "tester"]
}
```

Do you think that would work for your use case? You could pass in a custom element for each instance that you have on the site like this and supply any custom DOM you'd like to have in that element inside the grid.

**Comment by Acovendesigns** [comment]
Hey no problem, I'm loving using this thus far! Here's a WIP of what I'm
using it for  - https://alexcoven.com/type/

I think that would solve it, where were you thinking the HTML for the
custom element would be stored?

Cheers,
-Alex

<http://www.alexcoven.com> *Alex Coven*  /  Designer & Developer
portfolio  /  *alexcoven.com <http://www.alexcoven.com>*
phone  /  *216.409.1248 <(216)+409-1248>*
connect  /  *Dribbble <https://www.dribbble.com/alexcoven>  Instagram
<https://www.instagram.com/alcoven>  Behance
<https://www.behance.net/alcoven>*

**Comment by Acovendesigns** [comment]
Also another thing that would be really nice is having the ability to
change the init font size responsively or at 3 breakpoints I.E 400px 960px
and 1440px
Cheers,
-Alex

<http://www.alexcoven.com> *Alex Coven*  /  Designer & Developer
portfolio  /  *alexcoven.com <http://www.alexcoven.com>*
phone  /  *216.409.1248 <(216)+409-1248>*
connect  /  *Dribbble <https://www.dribbble.com/alexcoven>  Instagram
<https://www.instagram.com/alcoven>  Behance
<https://www.behance.net/alcoven>*

**Comment by kontur** [comment]
Cheers for the input, I've got something like that already noted in #12 — other things have priority, though.

**Comment by kontur** [comment]
>  I think that would solve it, where were you thinking the HTML for the custom element would be stored?

I imagine this HTML being placed anywhere on the page. You specify the element, and it will be "extracted" from where it is and added to where you specify in the `order` inside the Fontsampler instance. Does that make sense?

**Comment by kontur** [comment]
This is now released in `0.1.0`.
